torchgeo/datasets/biomassters.py

```diff
@@ -0,0 +1,344 @@
+            self._extract()
+            return
+
+        # Check if the user requested to download the dataset
+        if not self.download:
+            raise RuntimeError(
+                f"Dataset not found in `root={self.root}` and `download=False`, "
+                "either specify a different `root` directory or use `download=True` "
+                "to automatically download the dataset."
+            )
+
+        # Download the dataset
+        self._download()
+        self._extract()
+
+    def _download(self) -> None:
+        """Download the dataset and extract it.
+
+        Raises:
+            RuntimeError: if download doesn't work correctly or checksums don't match
+        """
+        # download features
+        download_url(
+            self.url.format(f"{self.split}_features.zip"),
+            self.root,
+            md5=self.md5s[f"{self.split}_features"] if self.checksum else None,
+        )
+        # download targets
+        if self.split == "train":
+            download_url(
+                self.url.format("train_agbm.zip"),
+                self.root,
+                md5=self.md5s["train_agbm"] if self.checksum else None,
+            )
+        # download metadata
+        download_url(self.url.format(self.metadata_filename), self.root)
+
+    def _extract(self) -> None:
+        """Extract the dataset."""
+        # extract imagery
+        extract_archive(os.path.join(self.root, f"{self.split}_features.zip"))
+        if self.split == "train":
+            extract_archive(os.path.join(self.root, "train_agbm.zip"))
+
+    def plot(
+        self,
+        sample: dict[str, Any],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample return by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional suptitle to use for figure
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+        """
+        ncols = len(self.sensors) + 1
+
+        showing_predictions = "prediction" in sample
+        if showing_predictions:
+            ncols += 1
+
+        fig, axs = plt.subplots(1, ncols=ncols, figsize=(5 * ncols, 10))
+        for idx, sens in enumerate(self.sensors):
+            img = sample[f"image_{sens}"].numpy()
+            if self.as_time_series:
+                # plot last time step
+                img = img[-1, ...]
+            if sens == "S2":
+                img = img[[2, 1, 0], ...]
+                img = percentile_normalization(img.transpose(1, 2, 0))
+            else:
+                co_polarization = img[0]  # transmit == receive
+                cross_polarization = img[1]  # transmit != receive
+                ratio = co_polarization / cross_polarization
+
+                # https://gis.stackexchange.com/a/400780/123758
```

Comment:
This only works for certain scales, see: https://github.com/microsoft/torchgeo/pull/1151#discussion_r1121153094

If you can't get it working, you can always plot them as separate grayscale images like I did for SSL4EO-S12. Maybe @RitwikGupta can help if he has time

Reply:
I assume this is still broken? We could fix it in a follow-up patch release I suppose.